### PR TITLE
Core v0.11

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/membrane_webrtc_plugin/endpoint_bin.ex [hunk 1]
```diff
@@ -351,7 +344,8 @@ defmodule Membrane.WebRTC.EndpointBin do
     encoding_specific_links =
       case encoding do
         :H264 when use_payloader? ->
-          &to(&1, {:h264_parser, ssrc}, %Membrane.H264.FFmpeg.Parser{alignment: :nal})
+          # &to(&1, {:h264_parser, ssrc}, %Membrane.H264.FFmpeg.Parser{alignment: :nal})
+          raise "Unsupported"
```
Comment (mickel8) [inline lib/membrane_webrtc_plugin/endpoint_bin.ex:348]: ?

File: mix.exs [hunk 2]
```diff
@@ -44,26 +44,29 @@ defmodule Membrane.WebRTC.Plugin.Mixfile do
...
-      {:membrane_ice_plugin, "~> 0.13.0"},
-      {:membrane_funnel_plugin, "~> 0.6.0"},
-      {:membrane_h264_ffmpeg_plugin, "~> 0.21.0"},
-      {:membrane_rtp_vp8_plugin, "~> 0.6.0"},
-      {:membrane_rtp_h264_plugin, "~> 0.13.0"},
-      {:membrane_rtp_opus_plugin, "~> 0.6.0"},
-      {:ex_libsrtp, "~> 0.6.0"},
+      # FIXME change after RTP bugfix release
+      # {:membrane_rtp_plugin, "~> 0.19.0"},
+      {:membrane_rtp_plugin, github: "membraneframework/membrane_rtp_plugin", override: true},
```
Comment (mickel8) [inline mix.exs:53]: Could we release this at first?

File: mix.exs [hunk 3]
```diff
@@ -44,26 +44,29 @@ defmodule Membrane.WebRTC.Plugin.Mixfile do
...
       {:ex_doc, "~> 0.28", only: :dev, runtime: false},
       {:dialyxir, "~> 1.1", only: :dev, runtime: false},
-      {:credo, "~> 1.6", only: :dev, runtime: false},
+      {:credo, "~> 1.6", only: :dev, runtime: false}
 
       # Otel
-      {:opentelemetry_api, "~> 1.0"},
-      {:opentelemetry, "~> 1.0.4"}
+      # {:opentelemetry_api, "~> 1.0"},
+      # {:opentelemetry, "~> 1.0.4"}
```
Comment (mickel8) [inline mix.exs:69]: Looks like you forgot to uncomment

File: lib/membrane_webrtc_plugin/endpoint_bin.ex [hunk 4]
```diff
@@ -677,52 +686,50 @@ defmodule Membrane.WebRTC.EndpointBin do
...
 
   @impl true
-  def handle_other({:signal, {:candidate, candidate}}, _ctx, state) do
+  def handle_parent_notification({:signal, {:candidate, candidate}}, _ctx, state) do
     candidate = "a=" <> candidate
 
     Membrane.OpenTelemetry.add_event(@ice_restart_span_id, :remote_candidate, candidate: candidate)
 
-    {{:ok, forward: {:ice, {:set_remote_candidate, candidate, 1}}}, state}
+    # {[notify_child: {:ice, {:set_remote_candidate, candidate, 1}}], state}
```
Comment (mickel8) [inline lib/membrane_webrtc_plugin/endpoint_bin.ex:708]: I believe that the candidate should be passed to ice and ignored there if it's not needed

**Review by mickel8 — CHANGES_REQUESTED** [review]

**Comment by daniel-jodlos** [comment]
I don't think I'm qualified to do a review of this plugin.

File: lib/membrane_webrtc_plugin/endpoint_bin.ex (hunk 4 above)
Comment (mat-hek) [reply lib/membrane_webrtc_plugin/endpoint_bin.ex]: Hmm, can we move that to a separate task? It seems these notifications in ICE should be cleaned up - for example, the WebRTC plugin handles `candidate_gathering_done` which is never sent and ICE claims it handles `peer_candidate_gathering_done` which is not true...

Comment (mickel8) [reply lib/membrane_webrtc_plugin/endpoint_bin.ex]: Sure

**Review by mickel8 — APPROVED** [review]

**Pull request merged**